nat-lab/tests/test_lana.py

```diff
@@ -482,6 +485,17 @@ def add_rtt_validators(
             )
 
 
+def add_nat_traversal_info_validator_to_non_symmetric_clients(
+    node_a_validator: EventValidator, node_a_pubkey: str, node_b_pubkey: str
+):
+    if base64.b64decode(node_a_pubkey) < base64.b64decode(node_b_pubkey):
```

Comment:
Just a confirmation, we only add the lower node validator because that will be initiator and will contain the NAT info ?

Reply:
That was a bit weird to me also, but as long as it makes decision between two pubkeys, I'm okay with that :smile:  [ref](https://github.com/NordSecurity/libtelio/blob/730cef24867b23ea84e3168290556f76abba775d/nat-lab/tests/test_direct_connection.py#L401)